kf/bpf.go

```diff
@@ -243,6 +253,19 @@ func (b *BPF) Stop(ifaceName, direction string, chain bool) error {
 	stats.SetWithVersion(0.0, stats.NFRunning, b.Program.Name, b.Program.Version, direction, ifaceName)
 
 	if len(b.Program.CmdStop) < 1 {
+		// Program ref handle - loaded from l3afd
+		if b.CollectionRef != nil {
+			if err := b.UnloadProgram(ifaceName, direction); err != nil {
+				return fmt.Errorf("BPFProgram %s unload failed with error: %v", b.Program.Name, err)
+			}
+			log.Info().Msgf("%s => %s direction => %s - program is unloaded/detached successfully", ifaceName, b.Program.Name, direction)
+			return nil
+		} else if len(b.Program.ObjectFile) > 0 {
```

Comment:
Will this ever get called?  When would we have an ObjectFile defined, but not a CollectionRef?  This would essentially mean we didn't perform a "LoadCollection" call